klog.go

```diff
@@ -697,6 +705,7 @@ func (l *loggingT) println(s severity, logr logr.Logger, args ...interface{}) {
 	}
 	fmt.Fprintln(buf, args...)
 	l.output(s, logr, buf, file, line, false)
+	l.fireHooks(s)
```

Comment:
Looks like fireHooks is always called after l.output, any chance we can move the call into `output`?

Reply:
But fireHooks and outPut are different things, should we tie them together?

Reply:
good point. let's leave it alone.